nemo_skills/pipeline/utils.py

```diff
@@ -684,6 +685,7 @@ def get_executor(
             # we need to be explicit about this in srun as commands might need to run in parallel
             f"--ntasks={tasks_per_node * num_nodes}",
             f"--nodes={num_nodes}",
+            f"--gpus-per-node={gpus_per_node}",
```

Comment:
Please add the same check as in line 679 here as some clusters don't support this parameter